openapi/components/schemas/PaymentCards/PaymentCard.yaml

```
@@ -1,126 +1,38 @@
-    readOnly: true
-  expYear:
-    description: Card's expiration year.
-    type: integer
-  expMonth:
-    description: Card's expiration month.
-    type: integer
-  billingAddress:
-    description: The Billing Address.
-    allOf:
-      - $ref: ../Contact/ContactObject.yaml
-  fingerprint:
-    description: >-
-      A unique value to identify the PAN regardless of expiration dates.  It
-      contains alphanumeric values.
-    type: string
-    readOnly: true
-  browserData:
-    $ref: ../PaymentTokens/BrowserData.yaml
-  panFingerprint:
-    deprecated: true
-    description: >-
-      A unique value to identify the PAN regardless of expiration dates.  It
-      contains alphanumeric values. Use `fingerprint` instead.
-    type: string
-    readOnly: true
-  status:
-    description: >
-      Payment Card status.  When a card is `active` it means it has been used at
-      least once for an approved transaction.
-
-      To remove a card from being in use, set it as `deactivated` (see the
-      deactivation endpoint).
-    type: string
-    readOnly: true
-    enum:
-      - active
-      - expired
-      - inactive
-      - deactivated
-      - pending
-      - verification-needed
-  brand:
-    readOnly: true
-    allOf:
-      - $ref: ./PaymentCardBrand.yaml
-  bankCountry:
-    description: Payment Card bank country
-    type: string
-    readOnly: true
-  bankName:
-    description: Payment Card bank name
-    type: string
-    readOnly: true
-  stickyGatewayAccountId:
-    description: Default Gateway Account ID used for transactions
-    type: string
-    readOnly: true
-  createdTime:
-    description: Card created time
-    allOf:
-      - $ref: ../ServerTimestamp.yaml
-  updatedTime:
-    description: Card updated time
-    allOf:
-      - $ref: ../ServerTimestamp.yaml
-  expirationReminderTime:
-    description: Time expiration reminder event will be triggered
-    nullable: true
-    allOf:
-      - $ref: ../ServerTimestamp.yaml
-  expirationReminderNumber:
-    description: Number of expiration reminder events triggered
-    type: integer
-    readOnly: true
-  customFields:
-    $ref: ../ResourceCustomFields.yaml
-  _links:
-    type: array
-    description: The links related to resource
-    readOnly: true
-    minItems: 1
-    items:
-      anyOf:
-        - $ref: ../Links/SelfLink.yaml
-        - $ref: ../Links/CustomerLink.yaml
-        - $ref: ../Links/AuthTransactionLink.yaml
-        - $ref: ../Links/ApprovalUrlLink.yaml
-  _embedded:
-    type: array
-    description: >-
-      Any embedded objects available that are requested by the `expand`
-      querystring parameter.
-    readOnly: true
-    minItems: 1
-    items:
-      anyOf:
-        - $ref: ../Embeds/AuthTransactionEmbed.yaml
-        - $ref: ../Embeds/CustomerEmbed.yaml
+allOf:
+  - $ref: ../Common/CommonPaymentCard.yaml
+  - properties:
+      stickyGatewayAccountId:
+        description: Default Gateway Account ID used for transactions.
```

Should it be an upper case? 
```suggestion
        description: Default gateway account ID used for transactions.
```